data/helpers.d/utils

```diff
@@ -1,5 +1,14 @@
 #!/bin/bash
 
+# Determine whether the scripts is executed from a terminal or the admin JS.
+#
+# [internal]
+ynh_is_term () {
+	# Return true if $TERM return xterm
+	# Which means the script is executed from a terminal, not from the admin JS
+	[[ "$TERM" =~ "xterm" ]]
```

Comment:
Uuuuh tested it on my side but for me it contains (during the execution of the script) `rxvt-unicode-256color`

Reply:
Huh, the name of your terminal is `unicode-256color` !?

Reply:
`rxvt-unicode-256color` yes, it's the whole `urxvt` thing ... but anyway it could be something else, e.g. I think gnome-terminal or other things have different values ?

Reply:
I don't think so actually, I think normal people using a normal terminal have xterm. Note that I'm not really using xterm, just the standard terminal of my distrib.

I'm going to try on my side as well, could you please try with `test -t` instead ?

Reply:
`test -t` says that the admin JS is a terminal...
I guess under the hood that's a virtual terminal which is being used...

Another solution is that apparently from the admin JS, I get `dumb` for $TERM, if you have the same, we can try with that.

Reply:
So in https://github.com/YunoHost/yunohost/pull/687/commits/95d2c8f79940feb8bf757bec48422bcc484b9253 I propose to add a new YNH_INTERFACE variable which is equal to `cli` or `api` depending of the use case.

 (I also removed the `ynh_is_term` helper because imho `[ $YNH_INTERFACE == "cli" ]` is semantic enough but we can keep the dedicated helper if you want, idk)

Reply:
Well, yes why not.
Seems a little overkill, but I believe it works.